staging/src/kubevirt.io/api/core/v1/types.go

```diff
@@ -2163,6 +2179,24 @@ type RemoveVolumeOptions struct {
 	DryRun []string `json:"dryRun,omitempty"`
 }
 
+// AddInterfaceOptions is provided when dynamically hot plugging a network interface
```

Comment:
Maybe I already asked it when reviewing the design (maybe not) I see that the binding type is not passed.
I guess that is because only bridge binding is supported. Won' it be problematic if we want in the future to extand the feature to any kind on binding (e.g sriov, macvtap)?

Reply:
Why would it be problematic ? It would be the default binding to be hotplugged - if more are supported, the user would have the opportunity to override it; if the user does not specify one, it'll default to use bridge binding.